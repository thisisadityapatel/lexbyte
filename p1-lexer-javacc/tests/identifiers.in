#Thisisanum #ttttHISisanothernum v_th1111110erer0er8 18?bbbbool123
?returning v_3iffy M_123Functional
v_thisisavec#Andanum?B42 M_12mat v_13vec
