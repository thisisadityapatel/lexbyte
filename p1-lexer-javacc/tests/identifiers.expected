String scanned: [#Thisisanum]
	->Token kind: <IDNUM>
	->Token value: "#Thisisanum"
String scanned: [#ttttHISisanothernum]
	->Token kind: <IDNUM>
	->Token value: "#ttttHISisanothernum"
String scanned: [v_th1111110erer0er8]
	->Token kind: <IDVEC>
	->Token value: "v_th1111110erer0er8"
String scanned: [18]
	->Token kind: <NUMBER>
	->Token value: 18
String scanned: [?bbbbool123]
	->Token kind: <IDBOOL>
	->Token value: "?bbbbool123"
String scanned: [?returning]
	->Token kind: <IDBOOL>
	->Token value: "?returning"
String scanned: [v_3iffy]
	->Token kind: <IDVEC>
	->Token value: "v_3iffy"
String scanned: [M_123Functional]
	->Token kind: <IDMAT>
	->Token value: "M_123Functional"
String scanned: [v_thisisavec]
	->Token kind: <IDVEC>
	->Token value: "v_thisisavec"
String scanned: [#Andanum]
	->Token kind: <IDNUM>
	->Token value: "#Andanum"
String scanned: [?B42]
	->Token kind: <IDBOOL>
	->Token value: "?B42"
String scanned: [M_12mat]
	->Token kind: <IDMAT>
	->Token value: "M_12mat"
String scanned: [v_13vec]
	->Token kind: <IDVEC>
	->Token value: "v_13vec"